Formula/convox.rb

```diff
@@ -16,7 +16,7 @@ class Convox < Formula
   def install
     ENV["GOPATH"] = buildpath
     (buildpath/"src/github.com/convox/rack").install Dir["*"]
-    system "go", "build", "-o", "#{bin}/convox", "-v", "github.com/convox/rack/cmd/convox"
+    system "go", "build", "-ldflags=-X main.Version=20160623185456", "-o", "#{bin}/convox", "-v", "github.com/convox/rack/cmd/convox"
```

Comment:
@DomT4 I think so, but I'll need to define that above, right? I originally had that, but it was removed by @MikeMcQuaid on https://github.com/Homebrew/homebrew-core/commit/072e6d1867e2b6ec81b2528a0cbebb7b5753d3e7 - I assume that was because it wasn't used anywhere at that point?